index.js

```diff
@@ -67,18 +67,20 @@ module.exports = {
             headers['Authorization'] = 'token ' + token;
           }
 
-          var options = {
-            method: 'POST',
-            uri: 'https://api.github.com/repos/' + org + '/' + repo + '/deployments',
-            headers: headers,
-            body: body,
-            json: true
+          console.log('willDeploy | token:', token);
+          let options = {
+            method: 'post',
+            url: `https://api.github.com/repos/${org}/${repo}/deployments`,
+            headers,
+            data: body,
           };
 
+          console.log('willDeploy | calling Github API', options, body);
```

Comment:
Try using `this.log(string, { verbose: true });` instead of `console.log`;

Reply:
Thanks! Unfortunately it didn't seem to have worked.

I should see it in the output of the relevant hook, like `willDeploy`, right?
<img width="291" alt="Screenshot 2022-01-25 at 18 05 40" src="https://user-images.githubusercontent.com/5022/151024457-0a8dd11a-024f-4ab7-b484-7a1a0e9e7300.png">

(Although searching for `willDeploy` in the CI output didn't turn up the `this.log` I added.)

Reply:
@balinterdi You didn't add the `{ verbose: true }` arg.